package_manifest.yaml

```diff
@@ -1,8 +1,16 @@
 ---
+all:
+  vars:
+    foremandist: .fm2_3
+    foreman_version: 'nightly'
+    katello_version: 'nightly'
+  children:
+    packages: {}
+    repoclosures: {}
+
 packages:
   vars:
     foremandist: .fm2_3
-    rpmlint_rc_file: "{{ inventory_dir }}/.rpmlintrc"
     foreman_version: 'nightly'
     katello_version: 'nightly'
```

Comment:
This looks the wrong way around: rpmlintrc should stay, but remove the moved ones?

Reply:
Yep, good catch.